index.html

```diff
@@ -8,13 +9,196 @@
+            <input data-category-id="21" type="button" name="sports" class="category-button text-capitalize"
+              value="sports">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="22" type="button" name="geography" class="category-button text-capitalize"
+              value="geography">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="23" type="button" name="history" class="category-button text-capitalize"
+              value="history">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="24" type="button" name="politics" class="category-button text-capitalize"
+              value="politics">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="25" type="button" name="art" class="category-button text-capitalize" value="art">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="26" type="button" name="celebrities" class="category-button text-capitalize"
+              value="celebrities">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="27" type="button" name="animals" class="category-button text-capitalize"
+              value="animals">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="28" type="button" name="vehicles" class="category-button text-capitalize"
+              value="vehicles">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="29" type="button" name="comics" class="category-button text-capitalize"
+              value="comics">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="30" type="button" name="gadgets" class="category-button text-capitalize"
+              value="gadgets">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="31" type="button" name="japanese anime &amp; manga"
+              class="category-button text-capitalize" value="japanese anime &amp; manga">
+          </div>
+          <div class="col-sm-full col-lg-half flex justify-center">
+            <input data-category-id="32" type="button" name="cartoon &amp; animations"
+              class="category-button text-capitalize" value="cartoon &amp; animations">
+          </div>
+          <div class="col-sm-full flex justify-center text-capitalize">
+            <input data-category-id="" type="button" name="i'm-crazy1" class="category-button text-capitalize"
```

Comment:
`i'm-crazy1`? Why 1? Why apostrophe? I would just make this name attribute's value `im-crazy`